static-site/gatsby-config.js

```diff
@@ -39,12 +39,6 @@ module.exports = {
       resolve: "gatsby-transformer-remark",
       options: {
         plugins: [
-          {
-            resolve: "gatsby-remark-images",
```

Comment:
I found these alternatives to [gatsby-remark-images](https://www.gatsbyjs.com/plugins/gatsby-remark-images/), which I'll look into later:

- [gatsby-remark-images-anywhere](https://www.gatsbyjs.com/plugins/gatsby-remark-images-anywhere/)
- [gatsby-remark-relative-images](https://www.gatsbyjs.com/plugins/gatsby-remark-relative-images/)

Reply:
Can't use either of those for this PR.

- [gatsby-remark-images-anywhere](https://www.gatsbyjs.com/plugins/gatsby-remark-images-anywhere/) requires gatsby-plugin-sharp.
- [gatsby-remark-relative-images](https://www.gatsbyjs.com/plugins/gatsby-remark-relative-images/) extends, not replaces, gatsby-remark-images.

Digging further, I don't think there is a way to serve Markdown images in Gatsby without `sharp`. All the different options in [this guide](https://www.gatsbyjs.com/docs/how-to/images-and-media/working-with-images-in-markdown/) end up using `gatsby-remark-images`/`gatsby-plugin-sharp`.

This is a dead end for the PR's intent to remove sharp-related dependencies.